src/utils/filenamehandler.cpp

```diff
@@ -24,6 +24,11 @@
 
 FileNameHandler::FileNameHandler(QObject *parent) : QObject(parent) {
     std::locale::global(std::locale(""));
+#ifdef Q_OS_WIN
+    invalidCharacters = QRegularExpression("[<>:\"/\\\\|?*]+");
```

Comment:
Because of the '+' at the end of the expression, consecutive invalid characters will be replaced by a single underscore. `:::` -> `_` for example. Not sure if that's a problem.

Reply:
Yep, I added precisely for that, otherwise `<*\` it will generate multiple `___`.

Reply:
@Rmano yeah, is hard to fix the problem in that kind of scenario, a user always has the option to manually copy the file generated in Linux/Unix to Windows, and in that case there's nothing the application can do.